api/src/api/schemas/search_schema.py

```diff
@@ -103,5 +108,51 @@ def with_one_of(
 
         return self
 
-    def build(self) -> Schema:
-        return BaseSearchSchema.from_dict(self.schema_fields, name=self.schema_class_name)  # type: ignore
+
+class DateSearchSchemaBuilder(BaseSearchSchemaBuilder):
+    """
+    Builder for setting up a filter for a date in the search endpoint schema.
+
+    Example of what this might look like:
+        {
+            "filters": {
+                "post_date": {
+                    "start_date": "YYYY-MM-DD",
+                    "end_date": "YYYY-MM-DD"
+                }
+            }
+        }
+
+    Support for start_date and
+    end_date filters have been partially implemented.
+
+    Usage::
+    # In a search request schema, you would use it like so:
+
+        example_start_date_field = fields.Nested(
+            StrSearchSchemaBuilder("ExampleStartDateFieldSchema")
```

Comment:
```suggestion
            DateSearchSchemaBuilder("ExampleStartDateFieldSchema")
```

as well as the below examples

Reply:
Ahh. Thanks for catching this oversight!